test/invariant-tests/handlers/AccessControlSender.Handler.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.9;
+
+/// library imports
+import "forge-std/Test.sol";
+
+/// local imports
+import {MessageSenderGAC} from "src/controllers/MessageSenderGAC.sol";
+import {MultiBridgeMessageSender} from "src/MultiBridgeMessageSender.sol";
+
+/// @notice handler for testing access control invariants
+contract AccessControlSenderHandler is Test {
+    /// @notice local state
+    MultiBridgeMessageSender public multiBridgeMessageSender;
+    MessageSenderGAC public gac;
+
+    /// @notice logs last caller for validations
+    address public lastCaller;
```

Comment:
`lastCaller` doesn't seem to be updated in this contract when the different functions are called